pkg/test/crdbuilder.go

```diff
@@ -78,15 +78,15 @@ func generateCRD(gvk schema.GroupVersionKind, scope apiextensionsv1.ResourceScop
 							Properties: map[string]apiextensionsv1.JSONSchemaProps{
 								"spec": {
 									Type:                   "object",
-									XPreserveUnknownFields: pointer.BoolPtr(true),
+									XPreserveUnknownFields: pointer.Bool(true),
 								},
 								"value": {
 									Type:                   "string",
-									XPreserveUnknownFields: pointer.BoolPtr(true),
+									XPreserveUnknownFields: pointer.Bool(true),
 								},
 								"status": {
 									Type:                   "object",
-									XPreserveUnknownFields: pointer.BoolPtr(true),
+									XPreserveUnknownFields: pointer.Bool(true),
```

Comment:
I can't see the relation with warfie, is this k8s package update required by the warfie bump? I am fine with the change and about upgrading dependencies in vendor folder, just curious about if all of them are triggered by warfie bump.

Reply:
Ah, this was probably because of the `k8s.io/api` bump and then the linter complained..